lib/client.go

```diff
@@ -86,3 +86,13 @@ func (c *Client) Query(ctx context.Context, collection string, conditions []*Con
 	}
 	return res, nil
 }
+
+// Add a document into a collection
+func (c *Client) Add(ctx context.Context, collection string, document interface{}) (*firestore.DocumentRef, error) {
```

Comment:
Could you use `Doc` type as a return type so that it can be parsed as JSON via `String()` func?

Reply:
Done